client/me/purchases/manage-purchase/index.jsx

```diff
@@ -599,10 +605,47 @@ class ManagePurchase extends Component {
 		return;
 	};
 
+	renderDisconnectedStateWarning() {
+		const { purchase, translate } = this.props;
+
+		const text = translate(
+			'This purchase is no longer connected with %(siteName)s and cannot be removed, please {{button}}contact support{{/button}}',
+			{
+				args: {
+					siteName: purchase.domain,
+				},
+				components: {
+					button: (
+						<button
+							className="purchase-item__link"
+							onClick={ ( event ) => {
+								event.stopPropagation();
+								event.preventDefault();
+								page( CALYPSO_CONTACT );
+							} }
+							title={ translate( 'Contact Support' ) }
+						/>
+					),
+				},
+			}
+		);
+
+		return (
+			<Card className="manage-purchase_disconnected" highlight="error">
```

Comment:
Minor, but `manage-purchase_disconnected` doesn't follow the [BEM pattern](https://css-tricks.com/bem-101/) of CSS class names. Since `disconnected` could be read as a modifier, this maybe could be `manage-purchase--disconnected`, but that would imply that this class is the entire "manage purchase" component, which it is not; this is a notice _about_ a disconnect _inside_ the manage purchase component. So a better className might be something like `manage-purchase__disconnected-notice` (the "block" is "manage-purchase" and the "element" is "disconnected-notice").

Reply:
Thanks for changing the CSS, but you also have to change it in the code.

Reply:
All set 👍